.build-tools/go.mod

```diff
@@ -18,3 +18,5 @@ require (
 	github.com/xeipuuv/gojsonreference v0.0.0-20180127040603-bd5ef7bd5415 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 )
+
+replace github.com/dapr/dapr => github.com/DeepanshuA/dapr v1.6.1-0.20230104160440-d2df8763f4c5
```

Comment:
Can this be removed or is it one of those changes that will break no matter what?

Reply:
This probably cannot be removed. These changes need to be made so that contrib can be merged.. and only after that can a PR be made against dapr/dapr which actually applies the changes from this fork. After that the replace statement can be removed.

This is very common today when updating component interfaces:
1. Update cert tests with dapr/dapr fork introducing future dapr/dapr changes to component interface
2. In dapr/dapr import latest contrib and make PR for component interface change
3. In contrib cert tests import dapr/dapr@master and remove replace statement.

And with pluggable components this gets even worse right now because before Step 2 you would also need to make a PR against the pluggable components SDK to apply the interface change.

Reply:
Thanks @berndverst for explaining. Yup, I can't remove this right now.